example/index.html

```diff
@@ -2,6 +2,7 @@
 <html lang="en">
   <head>
     <title>Notebook Demo</title>
+    <script type="text/javascript" src="https://cdn.mathjax.org/mathjax/latest/MathJax.js?config=TeX-AMS_HTML-full&delayStartupUntil=configured"></script>
```

Comment:
Ampersands in element attributes should be HTML encoded:

`https://cdn.mathjax.org/mathjax/latest/MathJax.js?config=TeX-AMS_HTML-full&amp;delayStartupUntil=configured`